nessie/src/main/java/org/apache/iceberg/nessie/NessieTableOperations.java

```diff
@@ -92,6 +92,11 @@ private TableMetadata loadTableMetadata(String metadataLocation, Reference refer
     // Nessie CLI will provide a reference aware GC functionality for the expired/unreferenced
     // files.
     newProperties.put(TableProperties.GC_ENABLED, "false");
+
+    newProperties.put(TableProperties.METADATA_DELETE_AFTER_COMMIT_ENABLED, "false");
+    LOG.info(
```

Comment:
This log message is meaningless, nothing actionable, I find it confusing.

Reply:
If the user has set the table property and might expect it to work. So, checking logs can help in analyzing why it didn't work. 

Do you think printing a warning log only when the user has set this property is useful?

Reply:
Added conditional warning logs and improved logs to contain actionable item.

Reply:
@snazy: We are waiting for your approval.